### PR TITLE
Grad doc

### PR DESCRIPTION
**Pull request by huangxu96**
### PR types

Bug fixes
### PR changes

Docs 
### Describe

Fix bugs in API Docs of grad.

### CODE REVIEW

File: python/paddle/fluid/dygraph/base.py [hunk 1]
```diff
@@ -479,10 +478,9 @@ def test_dygraph_grad(create_graph):
         .. code-block:: python
 
             import paddle
-            paddle.disable_static()
 
             def test_dygraph_grad(grad_outputs=None):
-                x = paddle.fluid.layers.fill_constant(shape=[1], value=2.0, dtype='float32')
+                x = grad_value = paddle.to_tensor(4.0)
```
Comment (TCChenlong) [inline python/paddle/fluid/dygraph/base.py]: 与中文不对应

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by wzzju — APPROVED** [review]
LGTM.

**Review by zhwesky2010 — COMMENTED** [review]
ltgm

**Review by zhwesky2010 — APPROVED** [review]
ltgm

**Pull request merged**